cvat-core/build.sh

```diff
@@ -0,0 +1,2 @@
+npm run build -- --mode=development
```

Comment:
Please remove the file.

Reply:
later... I do not want to copy files manually

Reply:
@ygnn123, you can keep them locally. No problems, just remove them from the pull request.